addons/hr_timesheet/models/project.py

```diff
@@ -291,8 +291,14 @@ def _compute_analytic_account_active(self):
 
     @api.depends('timesheet_ids.unit_amount')
     def _compute_effective_hours(self):
+        if self != self._origin:
```

Comment:
you can this trick : https://github.com/odoo/odoo/blob/14.0/addons/stock/models/stock_move.py#L320
`if not any(self._ids):`